pkg/chartutil/errors_test.go

```diff
@@ -0,0 +1,37 @@
+package chartutil
+
+/*
```

Comment:
typically, the copyright goes before the package declaration (see errors.go). Would you mind switching this around?

Reply:
Another whoops. I thought I had it at the top.